### ISSUE TITLE
Future comment layout change

### ISSUE THREAD

**Issue by gavin19** [honestbleeps/Reddit-Enhancement-Suite#1347]
http://www.reddit.com/r/modnews/comments/29i9yn/upcoming_change_cleanup_of_comment_markup/

It'll break a few things like keyboard nav/comment hide persistor/user tagger but they should be relatively easy to patch up.
